lib/rks/logger.rb

```diff
@@ -90,7 +90,7 @@ def with_rescue_and_duration_worker(correlation_id, worker, args, jid)
 private
   def mask_message(message)
     if message.is_a?(String)
-      message.gsub!(PATTERN, MASK)
+      message.gsub(PATTERN, MASK)
     elsif message.is_a?(Hash)
       message = JSON.dump(message)
       message.gsub!(PATTERN, MASK)
```

Comment:
Don't you need to change here as well?

Reply:
The problem is if the message is String, Its changed the original payload instead of change only the log. I think this issue will effect to any application that using rks.